### DISCUSSION TITLE
Does TopoLVM delete PVs when the ReclaimPolicy is set to Retain?

### DISCUSSION THREAD

**Discussion by Kavinraja-G** [topolvm/topolvm#845] [Q&A] [answered]
I would like to understand if TopoLVM deletes the PV when underlying node gets recycled even if the storageClass reclaimPolicy is also set to `Retain`?

This [doc](https://github.com/topolvm/topolvm/blob/main/docs/topolvm-controller.md#the-controller-for-nodes) indicates it deletes PVCs & LogicalVolumes when node gets removed, but there is also a limitation mentioned in this [doc](https://github.com/topolvm/topolvm/blob/main/docs/limitations.md#storageclass-reclaim-policy). Can anyone clarify this behaviour please?

**Comment by daichimukai** [comment, marked as answer]
> Does TopoLVM delete PVs when the ReclaimPolicy is set to Retain?

No, TopoLVM does not delete PV.
